tlatools/org.lamport.tlatools/src/tla2sany/drivers/SANY.java

```diff
@@ -90,14 +105,6 @@ public class SANY {
    * level-checking, and returns a Specification object, which is the
    * root of the semantic graph of the specification.
    *
-   * It also sends progress indications and a text error stream
-   * suitable for printing on System.out or System.err to the
-   * PrintStream "syserr" (unless "syserr" is null).  The error stream
-   * is also saved in Errors objects that are components of the
-   * Specification object returned.  The caller may prefer to pass
-   * syserr = null, and use the these Errors objects to provide
-   * feedback to the human spec-writer in some other way.
```

Comment:
Parts of this comment should be preserved. The bits about `Errors` are still true (I think?) and can be left here. The bit about printing to `syserr` belongs on the new `frontEndMain` override above.